2014-10-14 09:52:47,746 INFO: ERROR:404: IP:202.114.99.40 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 09:52:47,749 INFO: Header:Cookie: bdshare_firstime=1410233704254; CNZZDATA1000537224=305451716-1410233315-http%253A%252F%252Ftest.future.org.cn%252F%7C1413248515
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36
Connection: Keep-Alive
Via: 1.1 FUTURE40
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 12:16:27,367 INFO: ERROR:404: IP:202.114.114.2 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 12:16:27,368 INFO: Header:Cookie: __utma=19623079.2093875070.1350314847.1350314847.1350314847.1; bdshare_firstime=1363698941120; Hm_lvt_86bb9933fbc81638c6a43ece0fae26d4=1402379253,1402477616,1402477621,1402477622; Hm_lvt_997dd1ef1cd0ffc71b030c6fb102054c=1401637877,1401759807,1402227020,1402477613; CNZZDATA1000537224=169302819-1411200558-http%253A%252F%252Fwww.future.org.cn%252F%7C1413260263; arp_scroll_position=200
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36 SE 2.X MetaSr 1.0
Connection: Keep-Alive
Host: news.future.org.cn
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 14:07:13,530 INFO: ERROR:404: IP:202.114.71.112 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 14:07:13,531 INFO: Header:Cookie: bdshare_firstime=1369127765862; Hm_lvt_997dd1ef1cd0ffc71b030c6fb102054c=1398151553,1398167560,1398237905,1399211056; Hm_lvt_86bb9933fbc81638c6a43ece0fae26d4=1398252871,1398492143,1399211068,1399211081; CNZZDATA1000537224=320270805-1413266934-%7C1413266934
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36 SE 2.X MetaSr 1.0
Connection: Keep-Alive
Dnt: 1
Host: news.future.org.cn
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 14:45:12,417 INFO: ERROR:404: IP:202.114.99.40 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 14:45:12,418 INFO: Header:Cookie: bdshare_firstime=1410512783273; CNZZDATA1000537224=831930867-1410227513-http%253A%252F%252Fwww.future.org.cn%252F%7C1413269147
Content-Length: 
User-Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; Trident/4.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729)
Connection: Keep-Alive
Via: 1.1 FUTURE40
Host: news.future.org.cn
Accept: */*
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 16:55:21,314 INFO: ERROR:404: IP:58.215.136.23 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 16:55:21,315 INFO: Header:Clientip: 58.19.126.125
Cookie: PHPSESSID=b053a67051a6006b793b1e53657ec50a; bdshare_firstime=1400859981093
X-Piper-Id: 3739123222
Content-Length: 
Accept-Charset: x-gbk,utf-8;q=0.7,*;q=0.7
User-Agent: Mozilla/5.0 (Linux; U; Android 4.0.4; zh-CN; U705T Build/IMM76D) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 UCBrowser/9.9.5.489 U3/0.8.0 Mobile Safari/533.1
Connection: keep-alive
Host: news.future.org.cn
Cache-Control: max-age=259200
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8,UC/145,plugin/1,alipay/en
X-Forwarded-For: 58.19.126.125, 10.22.193.78
Accept-Language: zh-CN
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 17:07:59,874 INFO: ERROR:404: IP:220.181.125.129 REQUEST URL:http://news.future.org.cn/articles/54997.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 17:07:59,875 INFO: Header:If-Modified-Since: Fri, 05 Sep 2014 02:42:14 GMT
Content-Length: 
User-Agent: Sogou web spider/4.0(+http://www.sogou.com/docs/help/webmasters.htm#07)
Connection: close
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 17:44:56,945 INFO: ERROR:404: IP:219.222.64.112 REQUEST URL:http://news.future.org.cn/articles/54946.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 17:44:56,946 INFO: Header:Referer: http://zh.m.wikipedia.org/wiki/%E9%BB%84%E4%BE%83
Content-Length: 
User-Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 8_0_2 like Mac OS X; zh-CN) AppleWebKit/600.1.4.10.1 (KHTML, like Gecko) Mobile/12A405 UCBrowser/9.0.1.284 Mobile
Connection: keep-alive
Host: news.future.org.cn
X-Ucbrowser-Ua: pf(44);la(zh-CN);dv(iPh5,2);pr(UCBrowser);ov(8_0_2);pi(640x1136);ss(320x416);er(U);bt(UC);up();re(AppleWebKit/600.1.4.10.1 (KHTML, like Gecko));pm(0);bv(0);nm(0);im(0);nt(2);
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8,UC/44,ios_plugin/1
Accept-Language: zh-tw
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 17:58:30,734 INFO: ERROR:404: IP:180.149.130.18 REQUEST URL:http://news.future.org.cn/articles/73400.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 17:58:30,735 INFO: Header:Referer: http://image.baidu.com
Content-Length: 
Host: news.future.org.cn
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 18:52:01,448 INFO: ERROR:404: IP:10.132.6.213 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 18:52:01,449 INFO: Header:Cookie: CNZZDATA1000537224=1918621335-1413284026-%7C1413284026
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; Trident/7.0; rv:11.0) like Gecko
Connection: Keep-Alive
Dnt: 1
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 19:06:20,090 INFO: ERROR:404: IP:157.55.39.102 REQUEST URL:http://news.future.org.cn/articles/52483.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 19:06:20,091 INFO: Header:From: bingbot(at)microsoft.com
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)
Connection: Keep-Alive
Pragma: no-cache
Host: news.future.org.cn
Cache-Control: no-cache
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 19:09:20,274 INFO: ERROR:404: IP:27.17.143.196 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 19:09:20,275 INFO: Header:Cookie: bdshare_firstime=1363960933862; __utma=19623079.138606121.1368679033.1368679033.1378788418.2; lzstat_uv=22592162742841514395|3258983; CNZZDATA1000537224=270712671-1407648328-http%253A%252F%252Fwww.future.org.cn%252F%7C1413285065
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; MSIE 9.0; qdesk 2.4.1265.203; Windows NT 6.1; Trident/5.0)
Connection: Keep-Alive
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 21:43:05,225 INFO: ERROR:404: IP:68.180.228.95 REQUEST URL:http://news.future.org.cn/articles/73019.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 21:43:05,226 INFO: Header:Content-Length: 
User-Agent: Mozilla/5.0 (compatible; Yahoo! Slurp; http://help.yahoo.com/help/us/ysearch/slurp)
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, x-gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 22:05:34,022 INFO: ERROR:404: IP:157.55.39.241 REQUEST URL:http://news.future.org.cn/articles/50516.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 22:05:34,022 INFO: Header:From: bingbot(at)microsoft.com
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)
Connection: Keep-Alive
Pragma: no-cache
Host: news.future.org.cn
Cache-Control: no-cache
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 22:45:29,352 INFO: ERROR:404: IP:116.211.128.63 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 22:45:29,353 INFO: Header:Cookie: CNZZDATA1000537224=1507152351-1413297999-http%253A%252F%252Fwww.future.org.cn%252F%7C1413297999
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36 LBBROWSER
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 22:45:31,091 INFO: ERROR:404: IP:116.211.128.63 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 22:45:31,091 INFO: Header:Cookie: CNZZDATA1000537224=1507152351-1413297999-http%253A%252F%252Fwww.future.org.cn%252F%7C1413297999
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36 LBBROWSER
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 23:10:16,322 INFO: ERROR:404: IP:101.226.61.189 REQUEST URL:http://news.future.org.cn/article/ [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 23:10:16,322 INFO: Header:Content-Length: 
Accept-Charset: utf-8, iso-8859-1, utf-16, *;q=0.7
User-Agent: Mozilla/5.0 (Linux; U; Android 4.4.4; zh-tw; D5503 Build/14.4.A.0.108) AppleWebKit/533.1 (KHTML, like Gecko)Version/4.0 MQQBrowser/5.4 Mobile Safari/533.1 V1_AND_SQ_5.1.1_158_YYB_D QQ/5.1.1.2245
Connection: close
Host: news.future.org.cn
Accept: application/vnd.wap.xhtml+xml, text/vnd.wap.wml, application/xhtml+xml, text/html, image/png, image/jpeg, image/gif, */*;q=0.1
X-Forwarded-For: 113.57.254.62
Accept-Language: zh-TW
Content-Type: 
Accept-Encoding: gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-14 23:16:58,733 INFO: ERROR:404: IP:27.18.149.152 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-14 23:16:58,734 INFO: Header:Cookie: CNZZDATA1000537224=382509265-1413298134-%7C1413298134
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.124 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 06:23:03,411 INFO: ERROR:404: IP:157.55.39.110 REQUEST URL:http://news.future.org.cn/articles/16164.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 06:23:03,412 INFO: Header:From: bingbot(at)microsoft.com
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)
Connection: Keep-Alive
Pragma: no-cache
Host: news.future.org.cn
Cache-Control: no-cache
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 12:18:53,289 INFO: ERROR:404: IP:202.114.72.91 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 12:18:53,290 INFO: Header:Cookie: CNZZDATA1000537224=35678396-1413346836-%7C1413346836
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.102 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8,en;q=0.6
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 12:30:11,884 INFO: ERROR:404: IP:27.18.48.172 REQUEST URL:http://news.future.org.cn/articles/73385.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 12:30:11,884 INFO: Header:Referer: http://www.sogou.com/sogou?query=%CE%E4%BA%BA%B4%F3%D1%A7%CE%B4%C0%B4%CD%F8&pid=sogou-clse-7221e5c8ec6b08ef_3410
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36 SE 2.X MetaSr 1.0
Connection: Keep-Alive
Cookie: bdshare_firstime=1401007295682; Hm_lvt_997dd1ef1cd0ffc71b030c6fb102054c=1401116391; Hm_lvt_86bb9933fbc81638c6a43ece0fae26d4=1401116305,1401116471,1401116478,1401116522
Host: news.future.org.cn
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 12:31:41,039 INFO: ERROR:404: IP:202.114.72.91 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 12:31:41,040 INFO: Header:Cookie: CNZZDATA1000537224=35678396-1413346836-%7C1413346836
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.102 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8,en;q=0.6
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 12:33:44,568 INFO: ERROR:404: IP:116.211.131.25 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 12:33:44,569 INFO: Header:Cookie: CNZZDATA1000537224=951999727-1412653349-%7C1413347729
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0; MALNJS)
Connection: Keep-Alive
Dnt: 1
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 13:10:14,150 INFO: ERROR:404: IP:66.249.79.36 REQUEST URL:http://news.future.org.cn/lists/41-5.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 13:10:14,151 INFO: Header:From: googlebot(at)googlebot.com
Content-Length: 
User-Agent: Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)
Connection: Keep-alive
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 13:57:06,278 INFO: ERROR:404: IP:10.132.39.145 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 13:57:06,278 INFO: Header:Cookie: CNZZDATA1000537224=1579578805-1413041798-%7C1413352732
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko
Connection: Keep-Alive
Dnt: 1
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 16:06:16,990 INFO: ERROR:404: IP:210.42.117.100 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 16:06:16,990 INFO: Header:Content-Length: 
User-Agent: Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Trident/5.0; BIDUBrowser 2.x)
Connection: Keep-Alive
Host: news.future.org.cn
Cache-Control: no-cache
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 20:20:47,334 INFO: ERROR:404: IP:42.62.37.74 REQUEST URL:http://news.future.org.cn/articles/73625.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 20:20:47,335 INFO: Header:Content-Length: 
Accept-Charset: utf-8, gbk, gb2312, *;q=0.5
User-Agent: Mozilla/5.0 (compatible; ChinasoSpider; +http://shoulu.jike.com/spider.html)
Connection: close
Host: news.future.org.cn
Accept: text/html;q=0.8, */*;q=0.5
Accept-Language: zh-cn;q=0.8, *;q=0.5
Content-Type: 
Accept-Encoding: gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 20:35:13,381 INFO: ERROR:404: IP:115.156.110.218 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 20:35:13,383 INFO: Header:Cookie: bdshare_firstime=1408687676120; CNZZDATA1000537224=2140387449-1408687713-http%253A%252F%252Fwww.future.org.cn%252F%7C1413376169
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.3; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.137 Safari/537.36 LBBROWSER
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-15 22:00:17,295 INFO: ERROR:404: IP:220.181.125.129 REQUEST URL:http://news.future.org.cn/articles/39605.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-15 22:00:17,296 INFO: Header:If-Modified-Since: Wed, 13 Aug 2014 20:29:56 GMT
Content-Length: 
User-Agent: Sogou web spider/4.0(+http://www.sogou.com/docs/help/webmasters.htm#07)
Connection: close
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 04:27:47,695 INFO: ERROR:404: IP:117.27.158.79 REQUEST URL:http://news.future.org.cn/plus/mytag_js.php [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 04:27:47,696 INFO: Header:Content-Length: 
User-Agent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.0)
Host: news.future.org.cn
Cache-Control: no-cache
Accept: */*
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 07:17:40,807 INFO: ERROR:404: IP:123.150.183.123 REQUEST URL:http://news.future.org.cn/login/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 07:17:40,809 INFO: Header:Clientip: 113.57.182.8
Cookie: CNZZDATA1000537224=451758096-1413414142-http%253A%252F%252Fwww.future.org.cn%252F%7C1413414142
X-Piper-Id: 5350346464
Content-Length: 
Accept-Charset: x-gbk,utf-8;q=0.7,*;q=0.7
User-Agent: Mozilla/5.0 (Linux; U; Android 4.1.2; zh-CN; GN878 Build/JZO54K) AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 UCBrowser-CMCC/9.9.2.467 U3/0.8.0 Mobile Safari/533.1
Connection: keep-alive
Host: news.future.org.cn
Cache-Control: max-age=259200
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,text/vnd.wap.wml;q=0.6,*/*;q=0.5,UC/145,plugin/1,alipay/en
X-Forwarded-For: 113.57.182.8
Accept-Language: zh-CN
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 09:35:16,604 ERROR: Exception on /article/49926 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 954, in _execute_context
    context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 947, in _execute_context
    context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/default.py", line 435, in do_execute
    cursor.execute(statement, parameters)
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/cursors.py", line 205, in execute
    self.errorhandler(self, exc, value)
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorclass, errorvalue
OperationalError: (OperationalError) (2006, 'MySQL server has gone away') 'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' (49926, 1)
2014-10-16 09:35:18,271 ERROR: ERROR:500: IP:111.37.25.242 REQUEST URL:http://news.future.org.cn/article/49926 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 09:35:19,625 ERROR: Header:Referer: http://www.sogou.com/sogou?oq=%E6%96%B0%E7%94%9F%E6%B8%B8%E8%A7%88%E5%A4%A7%E5%AD%A6&stj0=0&query=%E6%96%B0%E7%94%9F%E6%B8%B8%E8%A7%88%E5%A4%A7%E5%AD%A6%E6%84%9F%E8%A7%A6&stj=0%3B0%3B0%3B0&stj2=0&pid=Af99046&duppid=1&stj1=0&hp=0&hp1=&sut=4009&ri=0&lkt=0%2C0%2C0&sst0=1413423316650&cid=&page=2&ie=utf8&p=50040111&dp=1&w=01025001&dr=1
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36 SE 2.X MetaSr 1.0
Connection: keep-alive
Host: news.future.org.cn
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 12:51:50,817 ERROR: Exception on /article/58248 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 891, in _execute_context
    None, None)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 885, in _execute_context
    conn = self._revalidate_connection()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 241, in _revalidate_connection
    "Can't reconnect until invalid "
StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) u'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' [immutabledict({})]
2014-10-16 12:51:51,026 ERROR: ERROR:500: IP:115.156.105.96 REQUEST URL:http://news.future.org.cn/article/58248 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 12:51:51,234 ERROR: Header:Cookie: CNZZDATA1000537224=1697685864-1412944804-%7C1413381883
Content-Length: 
User-Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; WOW64; Trident/4.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; InfoPath.3; .NET4.0C; .NET4.0E)
Connection: Keep-Alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 12:58:20,114 INFO: ERROR:404: IP:220.181.125.129 REQUEST URL:http://news.future.org.cn/articles/32552.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 12:58:20,115 INFO: Header:If-Modified-Since: Mon, 25 Aug 2014 23:25:38 GMT
Content-Length: 
User-Agent: Sogou web spider/4.0(+http://www.sogou.com/docs/help/webmasters.htm#07)
Connection: close
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 12:59:52,732 INFO: ERROR:404: IP:220.181.125.129 REQUEST URL:http://news.future.org.cn/articles/39690.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 12:59:52,733 INFO: Header:If-Modified-Since: Mon, 25 Aug 2014 23:26:30 GMT
Content-Length: 
User-Agent: Sogou web spider/4.0(+http://www.sogou.com/docs/help/webmasters.htm#07)
Connection: close
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip,deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 13:17:07,626 INFO: ERROR:404: IP:116.87.248.79 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 13:17:07,628 INFO: Header:Cookie: CNZZDATA1000537224=555368306-1410962332-http%253A%252F%252Fwww.future.org.cn%252F%7C1413436732
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.143 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 13:17:20,626 INFO: ERROR:404: IP:116.87.248.79 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 13:17:20,627 INFO: Header:Cookie: CNZZDATA1000537224=555368306-1410962332-http%253A%252F%252Fwww.future.org.cn%252F%7C1413436732
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.143 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 13:17:24,097 INFO: ERROR:404: IP:116.87.248.79 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 13:17:24,098 INFO: Header:Cookie: CNZZDATA1000537224=555368306-1410962332-http%253A%252F%252Fwww.future.org.cn%252F%7C1413436732
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/36.0.1985.143 Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8,en;q=0.6,zh-TW;q=0.4
Content-Type: 
Accept-Encoding: gzip,deflate,sdch

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 14:19:55,133 INFO: ERROR:404: IP:27.17.135.241 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 14:19:55,134 INFO: Header:Content-Length: 
User-Agent: Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.2; WOW64; Trident/6.0; BIDUBrowser 2.x)
Connection: Keep-Alive
Host: news.future.org.cn
Cache-Control: no-cache
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 14:41:35,923 INFO: ERROR:404: IP:36.63.1.97 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 14:41:35,925 INFO: Header:Cookie: bdshare_firstime=1399968584687; Hm_lvt_86bb9933fbc81638c6a43ece0fae26d4=1399968585; CNZZDATA1000537224=1823664897-1413441803-http%253A%252F%252Fwww.future.org.cn%252F%7C1413441803; Hm_lvt_997dd1ef1cd0ffc71b030c6fb102054c=1399968522
Content-Length: 
User-Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; .NET CLR 2.0.50727; .NET CLR 3.0.04506.648; Zune 4.7)
Connection: Keep-Alive
Host: news.future.org.cn
Accept: */*
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 14:43:43,372 ERROR: Exception on /article/58074 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 891, in _execute_context
    None, None)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 885, in _execute_context
    conn = self._revalidate_connection()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 241, in _revalidate_connection
    "Can't reconnect until invalid "
StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) u'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' [immutabledict({})]
2014-10-16 14:43:43,563 ERROR: ERROR:500: IP:220.181.108.172 REQUEST URL:http://news.future.org.cn/article/58074 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 14:43:43,756 ERROR: Header:Content-Length: 
User-Agent: Mozilla/5.0 (compatible; Baiduspider/2.0; +http://www.baidu.com/search/spider.html)
Connection: close
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-cn,zh-tw
Content-Type: 
Accept-Encoding: gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 14:44:17,818 ERROR: Exception on /article/58248 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 891, in _execute_context
    None, None)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 885, in _execute_context
    conn = self._revalidate_connection()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 241, in _revalidate_connection
    "Can't reconnect until invalid "
StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) u'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' [immutabledict({})]
2014-10-16 14:44:18,016 ERROR: ERROR:500: IP:36.63.1.97 REQUEST URL:http://news.future.org.cn/article/58248 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 14:44:18,212 ERROR: Header:Referer: http://news.future.org.cn/category/1
Content-Length: 
User-Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0; .NET CLR 2.0.50727; .NET CLR 3.0.04506.648; Zune 4.7)
Connection: Keep-Alive
Cookie: bdshare_firstime=1399968584687; Hm_lvt_86bb9933fbc81638c6a43ece0fae26d4=1399968585; CNZZDATA1000537224=1823664897-1413441803-http%253A%252F%252Fwww.future.org.cn%252F%7C1413441803; Hm_lvt_997dd1ef1cd0ffc71b030c6fb102054c=1399968522
Host: news.future.org.cn
Accept: image/gif, image/jpeg, image/pjpeg, image/pjpeg, application/msword, application/x-ms-application, application/x-ms-xbap, application/vnd.ms-xpsdocument, application/xaml+xml, */*
Accept-Language: zh-cn
Content-Type: 
Accept-Encoding: gzip, deflate

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 15:06:57,850 INFO: ERROR:404: IP:202.114.99.40 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 15:06:57,851 INFO: Header:Cookie: bdshare_firstime=1410233704254; CNZZDATA1000537224=305451716-1410233315-http%253A%252F%252Ftest.future.org.cn%252F%7C1413442533
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36
Connection: Keep-Alive
Via: 1.1 FUTURE40
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 15:07:00,625 ERROR: Exception on /article/58228 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 891, in _execute_context
    None, None)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 885, in _execute_context
    conn = self._revalidate_connection()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 241, in _revalidate_connection
    "Can't reconnect until invalid "
StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) u'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' [immutabledict({})]
2014-10-16 15:07:00,818 ERROR: ERROR:500: IP:202.114.99.40 REQUEST URL:http://news.future.org.cn/article/58228 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 15:07:01,020 ERROR: Header:Referer: http://news.future.org.cn/category/1
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36
Connection: Keep-Alive
Cookie: bdshare_firstime=1410233704254; CNZZDATA1000537224=305451716-1410233315-http%253A%252F%252Ftest.future.org.cn%252F%7C1413442533
Via: 1.1 FUTURE40
Host: news.future.org.cn
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 15:07:01,318 INFO: ERROR:404: IP:202.114.99.40 REQUEST URL:http://news.future.org.cn/favicon.ico [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 15:07:01,318 INFO: Header:Cookie: bdshare_firstime=1410233704254; CNZZDATA1000537224=305451716-1410233315-http%253A%252F%252Ftest.future.org.cn%252F%7C1413442533
Content-Length: 
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/31.0.1650.63 Safari/537.36
Connection: Keep-Alive
Via: 1.1 FUTURE40
Host: news.future.org.cn
Accept: */*
Accept-Language: zh-CN,zh;q=0.8
Content-Type: 

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
2014-10-16 15:44:55,303 ERROR: Exception on /article/58273 [GET] [in /usr/lib/python2.7/dist-packages/flask/app.py:1423]
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/future/wwwroot/news/news/controller/article.py", line 88, in viewfrontArticle
    article=db_article.Article().getArticleByid(id)
  File "/home/future/wwwroot/news/news/model/article.py", line 96, in getArticleByid
    return session.query(Article).filter(Article.id == id).first()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2334, in first
    ret = list(self[0:1])
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2201, in __getitem__
    return list(res)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2405, in __iter__
    return self._execute_and_instances(context)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/orm/query.py", line 2420, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 727, in execute
    return meth(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/sql/elements.py", line 322, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 824, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 891, in _execute_context
    None, None)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 1116, in _handle_dbapi_exception
    exc_info
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/util/compat.py", line 189, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 885, in _execute_context
    conn = self._revalidate_connection()
  File "/home/future/wwwroot/news/env/local/lib/python2.7/site-packages/sqlalchemy/engine/base.py", line 241, in _revalidate_connection
    "Can't reconnect until invalid "
StatementError: Can't reconnect until invalid transaction is rolled back (original cause: InvalidRequestError: Can't reconnect until invalid transaction is rolled back) u'SELECT news_article.id AS news_article_id, news_article.categoryid AS news_article_categoryid, news_article.title AS news_article_title, news_article.publisher AS news_article_publisher, news_article.author AS news_article_author, news_article.editor AS news_article_editor, news_article.`publishTime` AS `news_article_publishTime`, news_article.`updateTime` AS `news_article_updateTime`, news_article.content AS news_article_content, news_article.views AS news_article_views, news_article.state AS news_article_state, news_article.checker1 AS news_article_checker1, news_article.checker2 AS news_article_checker2 \nFROM news_article \nWHERE news_article.id = %s \n LIMIT %s' [immutabledict({})]
2014-10-16 15:44:56,738 ERROR: ERROR:500: IP:113.57.247.85 REQUEST URL:http://news.future.org.cn/article/58273 [in /home/future/wwwroot/news/news/controller/errorHandler.py:7]
2014-10-16 15:44:58,000 ERROR: Header:Q-Refer: 020200
Content-Length: 
Accept-Charset: utf-8, iso-8859-1, utf-16, *;q=0.7
User-Agent: Mozilla/5.0 (Linux; U; Android 4.4.2; zh-cn; NX505J Build/KVT49L) AppleWebKit/537.36 (KHTML, like Gecko)Version/4.0 MQQBrowser/5.3 Mobile Safari/537.36
Connection: keep-alive
Host: news.future.org.cn
Accept: application/vnd.wap.xhtml+xml, text/vnd.wap.wml, application/xhtml+xml, text/html, image/png, image/jpeg, image/gif, */*;q=0.1
Accept-Language: zh-CN
Content-Type: 
Accept-Encoding: gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:8]
2014-10-16 15:57:59,778 INFO: ERROR:404: IP:144.76.0.170 REQUEST URL:http://news.future.org.cn/articles/38309.html [in /home/future/wwwroot/news/news/controller/errorHandler.py:19]
2014-10-16 15:57:59,779 INFO: Header:Content-Length: 
User-Agent: Mozilla/5.0 (compatible; MJ12bot/v1.4.5; http://www.majestic12.co.uk/bot.php?+)
Connection: close
Host: news.future.org.cn
Accept: text/html,text/plain,text/xml,text/*,application/xml,application/xhtml+xml,application/rss+xml,application/atom+xml,application/rdf+xml
Accept-Language: zh
Content-Type: 
Accept-Encoding: gzip

 [in /home/future/wwwroot/news/news/controller/errorHandler.py:20]
